server/src/routes/routes.ts

```diff
@@ -1,9 +1,13 @@
 import { Router, Request, Response } from 'express';
+import ExpressWrapper from './ExpressWrapper';
+import getFilteredCars from '../controllers/cars/getFilteredCars';
 
 const router = Router();
 
 router.get('/', (req: Request, res: Response) => {
   res.send('hello');
 });
 
+router.get('/cars', ExpressWrapper(getFilteredCars));
```

Comment:
change file name to index.ts and here try to use all routes we have,
So create SubRoute for cars and here you can use it